deployment/base/nri-resmgr/daemonset.yaml

```
@@ -0,0 +1,79 @@
+kind: DaemonSet
```

missing apiVersion...
```suggestion
apiVersion: apps/v1
kind: DaemonSet
```

---

Fixed